src/steps/productContents/exteriorPaint.js

```diff
@@ -0,0 +1,130 @@
+export const extPaintContent = {
+  'which': {
+    buttons: [
+      {
+        text: "Stucco",
+        src: "http://www.aucklandroofing.net.nz/communities/0/004/009/612/140//images/4625618265.jpg",
+        buttonName: "stucco"
+      },
+      {
+        text: "Wood Siding",
+        src: "http://s3-production.bobvila.com/blogs/wp-content/uploads/2013/03/CertainTeed_Symphony-Composite-Slate-Roofing.jpg",
+        buttonName: "woodSiding"
+      }
+    ]
+  },
+  'dimensions': {
+    questions: [
```

Comment:
All the steps and the questions that is repeating here (from the roof), we can put in a separate file and it would be easier to menage them.